app/views/pictures/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="post-container">
```

Comment:
I'd try to avoid as much logic as possible from the view.

looks like you want to do something like:
```ruby
class Picture
  def liked_by?(user)
     ...
  end 
end
```